backend/src/main/java/com/huellapositiva/application/exception/UserNotConfirmed.java

```diff
@@ -0,0 +1,11 @@
+package com.huellapositiva.application.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.PRECONDITION_FAILED)
```

Comment:
This exception should not be necessary if you restrict actions to the roles that are CONFIRMED only